lib/breezystm32

```diff
@@ -1 +1 @@
-Subproject commit fb14d700999218e09a1f089847924324908766b9
+Subproject commit 3c404abcd6546fc19846bbe4a60011ebed7c3784
```

Comment:
So it looks like you've got an outdated BreezySTM32 reference that got committed. Not hard to fix, but I think that's happened a couple of times so maybe just try to keep an eye out for that. I'll go ahead and fix this one.